alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/transport/TransportExecuteMonitorAction.kt

```diff
@@ -93,7 +93,7 @@ class TransportExecuteMonitorAction @Inject constructor(
                     }
                 })
             } else {
-                val monitor = when (user == null || user?.name.isNullOrEmpty()) {
+                val monitor = when (user?.name.isNullOrEmpty()) {
```

Comment:
Interesting, I wasn't aware that user being null at the beginning would still call `isNullOrEmpty()` at the end of the call chain.